multithreading/consumers.py

```diff
@@ -43,20 +59,20 @@ def run(self):
             name, data = self.queue.get()
             self.lock.release()
 
-            try:
-                perform_check(name)(data)
-            except CriticalError as e:
-               self.gui.dispatch_alert("alert")
-               raise e  # this will cause emergency shutdown.
-            except WarningError as e:
-                self.gui.dispatch_alert("warning")
-                print(e)
-            except Exception as e:
-                print(e)
-                print(data)
+            if data.is_valid():
+                check_results = perform_check(name, data.value)
+                if isinstance(check_results, list):
```

Comment:
isinstance does not comply with coding standards, should be named is_instance or isInstance

Reply:
isinstance est une fonction native à Python